modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/util/comparator/UserNotificationEventUserNotificationEventIdComparator.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.notifications.web.internal.util.comparator;
+
+import com.liferay.portal.kernel.model.UserNotificationEvent;
+import com.liferay.portal.kernel.util.OrderByComparator;
+
+/**
+ * @author Roberto Díaz
+ */
+public class UserNotificationEventUserNotificationEventIdComparator
```

Comment:
Shouldn't this be named `UserNotificationEventUserNotificationEventIdOrderByComparator`? We almost always have to include the base class/iface as a suffix in class names.

Reply:
Sure, it's a typo.

I'll make the class name larger 😃